util/rlp-derive/src/de.rs

```diff
@@ -62,8 +66,7 @@ pub fn impl_decodable(ast: &syn::DeriveInput) -> TokenStream {
 	};
 
 	quote! {
-		#[allow(non_upper_case_globals, unused_attributes, unused_qualifications)]
-		const #dummy_const: () = {
+		const _: () = {
 			extern crate rlp;
```

Comment:
I guess that this can be removed.

Reply:
although `rlp_derive` crate uses 2018 edition, we cannot make this assumption about consumer crates (and it's actually not true)

Reply:
ok, I see